day2.md

```diff
@@ -0,0 +1,171 @@
+<img src="day2/14372101_1115754551807571_5092068918132506864_o.jpg" width="500"><br/>
+Like many legit restaurants we went to in Japan, the restaurant was pretty small and we ordered through a machine where you select what you want and put in the coins and it will give you tickets to give to a staff. <br />
+<img src="day2/IMG_20160906_183553.jpg" width="500"><br/>
+Not fully understood what all the words mean and feared of missing out, I ordered the basic ramen with a bunch of toppings. Which turned out to be way too much. Pro tip: just order the basic ramen, it comes with a bit of everything already.
+
+<img src="day2/IMG_20160906_184426.jpg" width="500"><br/>
+Look at the huge portion of noodles I got along with an extra egg, extra seaweeds and bamboo sheets.
+
+The broth was **NO JOKE**. The **richest/strongest** ramen broth I ever had.
+<img src="day2/14352284_1115754798474213_8765933718652016319_o.jpg" width="500"><br/>
+Look at how thick it is!
+
+I saw people first finished a half of the broth, then added water into the cup and finished the rest.
+
+The whole experience in Fuunji Ramen was wonderful and authentic: from waiting in line outside of the tiny restaurant, ordering through the machine, waiting in line inside seeing people eating in front of us, watching the chief cooking in front of us after we seated and gave us very warm smiles, to having the richest ramen broth in the world<sup>[4](#ramen)</sup>. I'm very glad that we went there and would say Fuunji Ramen(風雲児) is a must go if you want to have a nice authentic ramen experience! 😄
+
+We were up to ears when we left the restaurant. We then went to a fancy <a name="store">department store</a> where Matt fell in love with an long blue jean coat but did not get it because the price was a bit unreasonable<sup>[5](#coat)</sup>. We also rested in chairs at a Muji store for a while without shame before heading home. 😂
+
+-----
+<a name="algorithm">1</a>: I learned that, sadly, neither dfs nor bfs works in real life, since there's no playback nor fast-forward in life. Greedy algorithm, which does not guarantee to produce the most optimal solution, in many cases seems to be our best bet.
+
+<a name="pick">2</a>: To be honest, the main reason I picked it was because there were some photos of people, which seemed to be celebrities, eating at the restaurant hanging outside of it.
+
+<a name="small">3</a>: Which turned out to be somewhat true: throughout the trip, we noticed the patten that small restaurants that only offer one type of food were normally pretty darn good!
+
+<a name="ramen">4</a>: I probably did it the wrong way by eating it too fast or something, my tough was a bit numb after the meal and I did not finish the broth nor the noodles. 😂
+
+<a name="coat">5</a>: The remaining shopping parts in the trip consistently had one them for Matt: find the exact same coat in a more reasonable price.
```

Comment:
can it be "**for** a more reasonable price"? 
Why sometimes it's "for a price" sometimes it's "at a price"?